thehive/test/resources/data/Data.json

```diff
@@ -10,5 +10,9 @@
   {
     "id": "data-perdu.com",
     "data": "perdu.com"
+  },
```

Comment:
In the branch "develop-th4" Data.json is not used anymore. Data are included in Observable.json and entity Data are automatically created.